tools/locale-helper/index.js

```diff
@@ -0,0 +1,35 @@
+const translate = require('google-translate-api');
+const path = require('path');
+const fs = require('fs');
+
+const translationDir = path.resolve(__dirname, '../../app/translations');
+
+function writeJSON(file, data) {
+    const filePath = path.resolve(translationDir, file);
+    fs.writeFileSync(filePath, JSON.stringify(data, null, '\t'), 'utf8');
+}
+
+const { phrases } = require('../../app/translations/template');
+const supportedLangs = translate.languages;
```

Comment:
this is what we need to use here (we only support following languages)- 
```javascript
const supportedLangs = {
    "de": "Deutsch",
    "pl": "polski",
    "en": "English",
    "cs": "česky",
    "hi": "Hindi",
    "ru": "Русский",
    "co-id": "Indonesia",
    "bg": "български",
    "hu": "Magyar",
    "sr": "српски",
    "ml": "Malayalam",
    "ta": "தமிழ்",
    "nl": "Nederlands",
    "ja": "日本語",
    "pt": "Português",
    "codeans": "简体中文",
    "tr": "Türkçe",
    "codeant": "繁體中文",
    "es": "español",
    "ko": "한국어",
    "fr": "français"
};
```
put it in `support-lang.js` and export.